brave/brave-core
# Import web page to ipfs

Resolves https://github.com/brave/brave-browser/issues/14770

![image](https://user-images.githubusercontent.com/2965009/115279337-478f6400-a14f-11eb-81b6-817c02a82b1e.png)

- Added directory import
- Saving Web Page using "Save Page As" implementation
- The download item will be hidden in the download shelf
- Moved import files to separate import folder

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Check various cases when you have ongoing downloads and you do the import for a web page, it should not appear on the shelf with others
- Check that downloaded pages are not visible in the download history
- Check that the tab does not crash if downloading is in progress 
- Check imported files have same content as originals
- Check imported pages open correctly by shareable link
